src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -703,8 +703,11 @@ void RegisterAppInterfaceRequest::Run() {
   SendOnAppRegisteredNotificationToHMI(
       application, is_resumption_required && !is_resumption_failed_);
 
-  // By default app subscribed to CUSTOM_BUTTON
-  SendSubscribeCustomButtonNotification();
+  if (CheckHMICapabilities(mobile_apis::ButtonName::CUSTOM_BUTTON)) {
+    SDL_LOG_DEBUG("CUSTOM_BUTTON available");
+    SendSubscribeCustomButtonNotification();
```

Comment:
Please change name of this method because there is no SubscribeButtonNotifications for now

Reply:
fixed in
c778ad45111b62aa2ab237ea7120261a1b4a3f47